libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/headerSection.spec.ts

```diff
@@ -0,0 +1,95 @@
+// (C) 2023 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { Widget } from "../../tools/widget";
+import { EditMode } from "../../tools/editMode";
+import { InsightsCatalog } from "../../tools/insightsCatalog";
+import { LayoutRow } from "../../tools/layoutRow";
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+describe("Header section", { tags: ["pre-merge_isolated_tiger"] }, () => {
+    describe("Default language", () => {
+        beforeEach(() => {
+            cy.login();
+
+            Navigation.visit("dashboard/header");
+            new EditMode().isInEditMode();
+            new InsightsCatalog().waitForCatalogReload();
+        });
+
+        it("(SEPARATE) can update header for all sections", () => {
+            const section_01 = "  ";
+            const section_02 = "!@#$%^^&*";
+            const section_03 = "<button>click here</button>";
+
+            new LayoutRow(0).getHeader().setTitle(section_01).setDescription(section_01);
+            new LayoutRow(1)
+                .getHeader()
+                .setTitle("  " + section_02 + "  ")
+                .setDescription(section_02);
+            new LayoutRow(2)
+                .getHeader()
+                .setTitle(section_03)
+                .setDescription("  " + section_03 + "  ");
+
+            new EditMode().save();
+
+            new LayoutRow(0).getHeader().hasTitle(false).hasDescription(false);
+            new LayoutRow(1).getHeader().hasTitleWithText(section_02).hasDescriptionWithText(section_02);
+            new LayoutRow(2).getHeader().hasTitleWithText(section_03).hasDescriptionWithText(section_03);
+        });
+
+        it("Header is removed after latest insight is deleted from a section", () => {
+            new LayoutRow(0).hasCountOfHeaderSection(3);
+            new Widget(0).removeVizWidget();
+
+            new LayoutRow(0).hasCountOfHeaderSection(2);
+        });
+    });
+
+    describe("Localization", () => {
+        before(() => {
+            cy.login();
+
+            Navigation.visit("dashboard/header-localization");
+            cy.wait(10000); // need time to load localization config
+            new EditMode().isInEditMode();
+            new InsightsCatalog().waitForCatalogReload();
+        });
+        it("Header placeholder should be translated", () => {
+            const row_01 = new LayoutRow(0).getHeader();
+            row_01.setTitle(" ").setDescription(" ");
+
+            row_01
+                .hasPlaceholderTitle("Ajouter un titre ici...")
+                .hasPlaceholderDescription("Ajouter une description ici...");
+        });
+
+        it("(SEPARATE) limitation of title, description", () => {
+            cy.fixture("headerLimitTexts").then((data) => {
```

Comment:
As I mentioned above, I don't think we should use cy.fixture here because this case is not a data-driven test

Reply:
the object is too long. If define directly in .spec file, this makes the test file isn't clear and mess up